charts/kamaji/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: kamaji
 sources:
 - https://github.com/clastix/kamaji
 type: application
-version: 0.14.0
+version: 0.14.2
```

Comment:
I think this is the right version as the latest I previously pulled was `0.14.1`. Not sure if this is the only place to set the version.

Reply:
Our convention is minor bumping when adding a feature: if you don't mind, let's go for the 0.15.0.